packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -0,0 +1,234 @@
+var fs = require('fs');
+var path = require('path');
+var childProcess = require('child_process');
+var flowBinPath = require('flow-bin');
+const flowTypedPath = path.join(__dirname, 'node_modules', '.bin', 'flow-typed');
```

Comment:
I was actually going to ask the `flow-typed` team if I could do some work decoupling some of their stuff.  Would it be helpful if I brought this up?